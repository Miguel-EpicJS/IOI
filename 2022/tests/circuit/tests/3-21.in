385045b28fbc1903eca997d513040235e0229e01
363 882 5
-1 0 0 0 1 3 2 2 6 4 3 0 10 2 6 13 11 1 10 0 18 5 11 1 13 21 11 21 26 27 7 9 21 2 0 3 35 18 23 33 34 10 27 21 21 2 44 38 18 25 44 5 29 45 42 42 25 46 41 43 19 59 26 55 43 23 11 59 57 62 1 27 67 55 73 34 58 63 64 54 4 47 23 32 5 69 47 23 49 4 33 56 82 87 68 65 58 64 0 48 7 32 35 63 49 98 90 88 37 105 104 110 82 88 90 86 109 32 94 101 61 88 101 101 61 121 119 80 83 43 126 74 34 109 128 62 83 116 59 135 121 117 77 109 117 129 66 98 145 143 114 116 61 144 83 126 146 141 156 105 130 137 136 144 148 155 156 154 134 155 45 168 3 156 159 61 169 153 128 165 156 144 82 182 43 127 182 127 126 98 184 172 136 100 183 172 184 182 137 134 199 195 142 187 200 168 155 166 181 142 170 206 188 80 183 159 199 143 136 167 183 203 197 182 209 182 196 222 165 224 104 193 199 210 199 179 223 170 215 88 172 237 172 238 161 201 181 189 213 232 191 245 250 112 151 196 255 123 225 129 249 240 215 215 195 184 178 234 146 176 246 258 264 270 159 272 246 184 213 246 259 244 257 257 185 262 268 272 259 161 256 165 241 169 242 274 284 284 169 258 230 195 296 260 283 299 265 296 305 256 258 202 211 251 306 277 279 237 306 228 193 255 264 313 315 276 315 314 304 311 241 276 325 239 241 317 322 313 316 330 337 327 266 336 276 261 130 345 344 335 317 331 350 331 322 344 158 353 266 330 356 344 361 5 1 15 12 5 5 16 18 13 12 16 6 21 13 14 12 15 23 25 15 25 29 28 31 16 8 24 24 9 20 20 14 34 24 22 35 36 32 41 20 33 38 23 36 39 39 30 14 43 17 39 7 31 50 28 52 49 51 30 34 52 6 38 20 18 19 56 53 56 48 46 53 53 66 35 40 36 57 44 70 33 53 63 22 63 45 51 55 42 10 52 57 73 40 64 67 76 70 59 17 31 51 70 8 37 51 72 74 22 26 77 67 70 84 71 38 64 74 50 60 50 84 85 89 81 49 72 42 79 92 65 8 87 76 35 70 71 88 76 41 92 68 71 47 78 87 77 73 29 48 78 97 60 78 75 85 75 100 54 83 28 9 106 103 106 59 107 99 44 69 56 81 54 62 25 95 91 46 37 108 111 101 91 79 110 110 17 79 108 106 119 81 107 56 114 30 113 45 114 102 94 116 83 90 72 16 114 93 80 101 93 86 97 91 86 120 96 123 119 103 63 120 122 7 124 129 88 60 41 69 119 114 112 130 19 70 129 85 93 100 133 118 113 100 107 130 75 135 103 117 122 135 90 112 115 67 138 96 133 111 134 111 138 34 58 138 68 96 102 78 66 129 133 117 141 49 131 105 119 146 123 140 131 95 145 150 142 59 141 153 150 113 133 145 141 148 105 128 127 147 140 104 124 78 159 132 77 120 163 150 131 166 139 157 99 171 162 147 152 139 121 164 97 122 172 80 99 152 171 178 167 109 160 177 143 122 115 147 89 157 163 167 179 89 158 84 183 92 102 173 171 173 174 174 149 177 162 189 152 138 156 152 188 125 115 180 140 40 151 139 192 147 154 191 157 197 175 74 176 192 199 192 204 129 187 196 208 180 95 195 149 174 173 200 210 197 188 166 209 206 203 198 148 175 151 65 185 179 132 125 214 202 170 164 190 217 211 216 198 161 204 178 195 206 158 212 200 221 205 186 222 209 193 219 223 214 189 118 168 190 177 217 232 221 218 190 234 225 185 233 210 228 149 182 213 232 221 203 228 225 222 207 164 90 214 99 164 216 118 226 94 187 154 228 220 241 227 242 244 162 201 159 125 194 198 250 226 181 229 246 243 244 235 239 240 249 254 238 253 252 257 240 231 236 205 247 228 253 251 238 249 227 257 186 207 143 248 219 201 254 242 262 265 140 253 216 180 243 247 243 242 230 260 268 176 263 124 248 234 219 190 186 211 229 263 236 261 224 269 230 223 261 192 194 258 153 231 276 250 277 274 217 154 287 163 206 72 271 275 233 280 175 223 204 196 269 292 218 294 284 286 235 283 237 202 287 288 283 274 285 278 267 155 254 289 273 291 296 297 235 300 272 248 226 267 137 286 277 227 301 218 282 255 207 243 225 160 299 299 281 295 301 295 252 270 108 288 160 289 292 234 272 298 236 295 288 208 281 293 297 191 282 301 208 282 290 259 260 132 293 262 280 224 294 273 287 307 309 292 271 246 212 308 289 312 304 292 245 306 320 275 316 285 317 265 269 279 290 318 258 194 321 308 315 278 295 298 212 310 251 310 278 263 305 310 205 256 312 307 324 326 320 323 247 303 300 312 300 245 303 185 318 229 321 244 316 307 319 325 307 326 330 297 317 267 320 286 296 321 322 302 331 327 290 308 275 318 268 311 239 285 311 329 309 324 323 294 329 333 334 314 332 330 328 334 324 231 326 333 252 280 309 329 328 260 305 331 220 311 281 336 331 338 266 332 336 304 291 341 298 339 340 314 328 277 333 302 271 343 233 343 335 279 342 338 327 319 334 323 342 345 317 325 340 346 291 337 340 346 264 342 332 220 313 347 345 306 293 343 338 339 349 349 319 346 341 303 339 277 351 341 348 346 273 302 352 270 353 351 333 350 342 347 355 354 353 348 354 355 352 349 347 358 357 358 354 359 348 352 332 346 357 359 350 357 361 360 356 360 355 335 337 362 359 362 351 362 358 358 360 356 361
1 0 1 0 1 1 0 0 1 1 0 0 0 1 1 0 1 0 0 1 0 1 0 1 0 0 1 0 1 0 1 0 0 1 0 1 1 0 0 1 1 0 0 0 1 0 1 0 0 1 1 1 1 1 1 0 1 1 1 1 0 0 0 1 1 1 0 0 0 1 0 1 0 1 1 0 0 1 0 1 0 0 0 0 1 0 0 0 0 0 0 1 0 1 0 0 0 0 0 0 0 0 1 0 1 1 1 0 0 0 0 1 1 0 0 0 1 1 0 0 1 0 1 1 1 1 0 0 0 0 1 1 1 0 1 0 1 1 0 1 0 1 1 0 0 1 0 1 0 0 0 1 1 0 0 0 0 1 1 1 0 0 0 0 0 1 1 0 1 1 0 0 1 1 0 0 1 0 1 0 0 0 0 1 0 0 1 0 0 0 0 1 1 0 0 0 1 0 0 1 1 0 1 1 1 1 1 1 0 0 1 0 0 0 1 1 0 0 0 0 1 0 0 0 1 0 0 1 1 1 1 1 1 1 0 0 0 0 0 0 0 1 0 1 1 1 1 0 1 0 1 1 1 0 0 0 1 1 1 1 0 0 1 0 0 1 0 1 0 1 1 1 1 1 0 1 0 0 1 1 0 1 0 0 0 0 1 1 1 1 0 1 0 0 1 1 1 1 0 0 0 0 1 1 0 0 0 1 1 1 1 0 0 1 0 1 0 0 0 0 0 0 1 0 0 0 0 0 0 0 1 1 1 0 0 1 0 1 0 1 0 0 0 0 1 0 1 1 0 1 0 1 1 1 1 1 0 1 0 0 0 1 0 1 0 1 1 1 1 1 1 0 1 1 1 0 0 0 0 1 1 0 0 1 1 0 0 0 0 0 0 1 1 0 0 0 1 0 0 0 0 0 0 1 0 1 1 0 0 0 1 0 1 1 0 0 1 0 0 1 1 0 0 0 0 0 1 1 0 0 1 1 1 0 1 0 1 0 1 1 1 0 0 1 0 0 1 0 1 1 1 1 0 1 0 1 0 0 1 1 1 1 0 1 0 1 1 1 1 1 0 1 1 0 0 0 0 0 1 0 1 1 1 1 0 0 0 0 0 0 1 0 0 0 0 0 1 1 1 1 1 1 0 0 0 0 0 0 1 0 1 0 0 1 0 1 1 1 1 0 0 0 0 1 1 0 1 1 1 1 1 1 0 1 1 1 1 0 1 0 1 0 0 1 1 0 0 1 0 1 1 0 1 0 1 1 1 0 0 0 0 0 1 1 1 1 1 1 0 1 0 0 1 0 1 0 0 0 1 0 0 0 1 1 0 0 1 1 1 0 1 1 0 0 0 0 0 0 0 1 1 0 0 0 1 0 1 0 0 1 0 0 0 1 0 0 0 0 1 1 0 0 1 0 1 0 1 1 1 0 0 1 1 0 0 0 1 1 1 0 1 1 0 1 1 1 0 0 1 1 1 0 0 0 1 0 1 0 0 0 1 1 0 0 1 1 1 1 0 1 0 1 0 1 1 1 1 1 1 1 1 1 0 0 0 0 0 0 1 1 1 1 0 0 0 0 0 0 0 1 0 1 0 1 0 1 0 0 0 1 0 1 0 1 1 1 1 1 1 0 0 1 1 1 0 0 0 1 1 1 0 0 0 1 1 1 0 1 1 1 0 1 1 0 0 1 1 1 0 1 0 1 1 1 1 1 0 1 1 1 0 0 1 0 1 0 1 0 0 1 1 0 0 0 1 1 0 1 0 1 1 1 0 0 0 1 0 0 1 0 0 1 0 1 1 1 1 0 0 0 0 1 1 1 0 1 0 1 1 0 1 1 0 1 1 0 1 1 1 1 0 1 1 1 1 1 0 0 1 1 0 0 0 0 1 1 0 0 0 1 1 1 0 0 1 0 1 1 1 0 1 0 0 1 0 1 1 0 0 0 0 1 0 0 1 0 1 0 1 0 0 1 1 1 1 0 1 0 1 0 1 1
441 620
441 1012
363 1147
530 571
810 1018
