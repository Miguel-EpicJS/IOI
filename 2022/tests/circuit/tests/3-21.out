4faa78c22ee35083e49b65601698a1bd98a87a0f
OK
860608712
828138786
455393642
548429080
972077788
